src/components/select/filterable-select/filterable-select.stories.mdx

```diff
@@ -210,6 +210,59 @@ A custom `Button` Component could be passed as the `listActionButton` value.
   </Story>
 </Preview>
 
+### Asynchronous:
+
```

Comment:
Please an we add a text description explaining when the use this pattern. There's always a lot of confusion around lazy-loading/infinite loading.

We should make it very clear which use case it is that we're supporting. I assume this is only for loading the items on open of dropdown, but not to load more options when the input changes? Equally, it shouldn't be used to load more items when the user scrolls?

Whatever the intended usage is, we should document it, please do the same for the simple story.

Reply:
right now the lazy loading is not yet implemented, it's part of another task. When the other task done I'll add a description on which one should be used.